cmd/payload-testing-ui/server.go

```diff
@@ -48,6 +46,9 @@ const (
 <h1>{{ .ObjectMeta.Namespace }}/{{ .ObjectMeta.Name }}</h1>
 
 <p class="text-right">{{ returnLink }}</p>
+<p class="text-right">{{ documentationLink }}</p>
```

Comment:
These two links look out of place in the page layout, we can use a navigation bar similar to the one in https://steps.ci.openshift.org for them.